File: common/src/test/java/com/hartwig/actin/common/DatamodelPrinterTest.java
package com.hartwig.actin.common;

import org.junit.Test;

public class DatamodelPrinterTest {

    @Test
    public void canPrintWithVaryingIndents() {
        for (int i = 0; i < 10; i++) {
            DatamodelPrinter printer = new DatamodelPrinter(i);
            printer.print("hi");
        }
    }
}

File: common/src/test/java/com/hartwig/actin/util/DatamodelPrinterTest.kt
package com.hartwig.actin.util

import org.junit.Test

class DatamodelPrinterTest {
    @Test
    fun canPrintWithVaryingIndents() {
        for (i in 0..9) {
            val printer = DatamodelPrinter(i)
            printer.print("hi")
        }
    }
}